requirements.txt

```diff
@@ -4,31 +4,33 @@
 #
 #    pip-compile requirements.in
 #
-alembic==1.7.4
+alembic==1.7.7
     # via flask-migrate
-amqp==5.0.9
+amqp==5.1.1
     # via kombu
 arrow==1.2.2
     # via isoduration
-attrs==21.2.0
+async-timeout==4.0.2
+    # via redis
+attrs==21.4.0
     # via jsonschema
-awscli==1.21.4
+awscli==1.24.8
     # via awscli-cwlogs
 awscli-cwlogs==1.4.6
     # via -r requirements.in
-bcrypt==3.2.0
+bcrypt==3.2.2
     # via flask-bcrypt
 beautifulsoup4==4.11.1
     # via -r requirements.in
 billiard==3.6.4.0
     # via celery
-bleach==4.1.0
+bleach==5.0.0
```

Comment:
there's a few breaking changes here (https://bleach.readthedocs.io/en/latest/changes.html)

i dont think this'll impact us - but could there be some weird potential for issues if someone passes in a html element with style tags? that's weird and unlikely but i dont want us to be open to bugs because of it (cc @quis)

Reply:
I've rebased this out so that it can be looked at separately, so there's now no change to the bleach version